tests/utils.py

```diff
@@ -35,14 +35,46 @@ def run_stage(project_path, project_parameters_file_name="ProjectParameters.json
 
         return model, stage
 
+    @staticmethod
+    def run_stages(project_path: Union[str, Path], project_parameters_files: List[Union[str, Path]]):
+        """
+        Runs multiple stages of the calculation.
+
+        Args:
+            - project_path (Union[str, Path]): The path to the project folder.
+            - project_parameters_files (List[Union[str, Path]]): List of project parameters files.
+
+        """
+
+        cwd = os.getcwd()
+
+        # initialize model
+        model = Kratos.Model()
+
+        # loop over all stages
+        for file_name in project_parameters_files:
+            # change working directory to test file directory
+            os.chdir(project_path)
+
+            # read parameters
+            with open(file_name, 'r') as parameter_file:
+                parameters = Kratos.Parameters(parameter_file.read())
+
+            # run stage
+            stage = analysis.StemGeoMechanicsAnalysis(model, parameters)
+            stage.Run()
+
+            # change working directory back to original working directory
+            os.chdir(cwd)
```

Comment:
why is run_stage returning and run_multiple_stages not returning? 
Also both functions could be merged. You iterate around project_parameters_files. In the case of one stage this is one loop